src/Scene.cpp

```diff
@@ -38,26 +39,54 @@ const std::vector<PointLight>& Scene::getPointLights() const {
 // Camera movement
 
 void Scene::moveCameraLeft() {
-  cameraPos.x += 0.05;
-  cameraTarget.x += 0.05;
+  float PI_BY_2 = glm::pi<float>() / 2.0;
+  glm::vec3 dir = glm::normalize(cameraTarget - cameraPos);
+  dir = glm::rotateY(dir, PI_BY_2) * stepSize;
+  cameraTarget += dir;
```

Comment:
I think we may want to just update the x and z positions, if we're rotating about y